app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ActiveRecord::Base
+  belongs_to :post
+  belongs_to :user
+  has_many :likes, as: :likeable
+
+  def already_liked_by?(current_user)
```

Comment:
This really gets me curious: you have this method in both Comment AND Post.

I wonder if there's a way to define such a helper method in both classes in one spot? (I don't know if there is.)

Say we have another class that comment and post can inherit from.